package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.5",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
```

Comment:
- [ ] When I start the front-end app I get the below error, Please fix this issue and make sure the app works without any issues.
  - To fix this issue you should run this command in the terminal. `npm install react-router-dom`

![image](https://user-images.githubusercontent.com/83803414/208500888-9cc193ad-79aa-4461-84d9-fbe4ca11cea5.png)